.github/workflows/aesni.yml

```diff
@@ -15,6 +15,8 @@ defaults:
 env:
   CARGO_INCREMENTAL: 0
   RUSTFLAGS: "-Dwarnings"
+  RUSTDOCFLAGS: "-C target-feature=+aes,+ssse3"
+  RUSTFLAGS: "-C target-feature=+aes,+ssse3"
```

Comment:
It looks like `RUSTFLAGS` is duplicated here? I'm not sure how GitHub actions handles that